arch/arm/configs/bcm2711_defconfig

```diff
@@ -890,6 +890,7 @@ CONFIG_DRM_PANEL_SIMPLE=m
 CONFIG_DRM_PANEL_RASPBERRYPI_TOUCHSCREEN=m
 CONFIG_DRM_V3D=m
 CONFIG_DRM_VC4=m
+CONFIG_DRM_VC4_HDMI_CEC=y
```

Comment:
Is this not also applicable to 64-bit builds?

Reply:
Untested but no reason I can think of why 64-bit build would fail.

Reply:
Throw it in there, then.